lib/main.js

```diff
@@ -18,7 +20,11 @@ Object.assign(exports, {
     PlacesProvider.links.init();
     SearchProvider.search.init();
     options.telemetry = false;
-    app = new ActivityStreams(options);
+
+    Task.spawn(function*() {
+      options.clientID = yield ClientID.getClientID();
```

Comment:
@vladikoff suggested to use `identity.fxaccounts.lastSignedInUserHash`. He will confirm it.

We can use `ClientID.getClientID()` as a fallback in case the user is not signed in to FxA.